src/planner.rs

```diff
@@ -171,13 +271,32 @@ impl<C: 'static> Planner<C> {
     }
 
     /// Runs a custom system.
+    ///
+    /// ## Deprecated
+    ///
+    /// This method is deprecated,
+    /// because it blocks until
+    /// all the other systems are finished.
+    ///
+    /// Please use [`Planner::add_system`]
+    /// instead.
+    ///
+    /// [`Planner::add_system`]: struct.Planner.html#method.add_system
+    #[deprecated(note = "Add a system instead")]
```

Comment:
I don't think we should deprecate `run_custom`, because you can call it in between calls to `dispatch`, for example, if you wanted to do the `run_custom` while no other system is running (which dispatch doesn't guarantee). Also it's pretty useful for shorter demonstrations and examples.